pkgs/development/python-modules/pdftotext/default.nix

```
@@ -9,6 +9,11 @@ buildPythonPackage rec {
     sha256 = "a067c121654917ecbe07fbd71c807c34bbdb1ea029e269ddd11925ee7e191d3f";
   };
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace setup.py \
+        --replace '= brew_poppler_include()' '= "${poppler}/include", "${poppler}/lib"'
```

Just to make it a bit more multi-output resistant 
```suggestion
        --replace '= brew_poppler_include()' '= "${lib.getDev poppler}/include", "${lib.getLib poppler}/lib"'
```

---

Thanks, I didn't know about those helper functions.

Updated in latest force-push.